pokemon-league-participation-manager/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/common/PersistenceApplicationContext.java

```diff
@@ -30,7 +29,6 @@
      *
      * @return Transactional manager
      */
-    @Bean
```

Comment:
Are we sure @ Bean is not needed here?

Reply:
I was unclear - here was meant for all methods in this class. Since @ Bean lets spring know that a method is returning a Bean. In the issue, I only requested the nuking of the method that returns the validator bean and no change to the others.

Reply:
So it was a misunderstanding, as I get it right it should be this way, right? (Same as in the sample project)

Reply:
Yes, that is correct.